scripts/dts/edtlib.py

```diff
@@ -416,7 +416,8 @@ def _check_binding(self, binding, binding_path):
                 _err("missing, malformed, or empty '{}' in {}"
                      .format(prop, binding_path))
 
-        ok_top = {"title", "description", "compatible", "properties", "#cells",
+        ok_top = {"title", "description", "compatible", "variant-compatible",
+                  "properties", "#cells",
```

Comment:
Could reflow all of `ok_top`, or put on separate lines if you're worried about history (though I wonder if the vertical spam is worthwhile).